server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -109,6 +109,10 @@ object LivyConf {
   // How often Livy polls YARN to refresh YARN app state.
   val YARN_POLL_INTERVAL = Entry("livy.server.yarn.poll-interval", "5s")
 
+  // Http compression is enabled or not, if true the json data transmitted with be
+  // compressed with GZIP.
+  val COMPRESS_SUPPORTED = Entry("livy.server.content_compress.enabled", false)
```

Comment:
From my understanding, several http clients like python's requester and apache http client as well as Chrome web browser by default enable gzip in their http request's accept-header, so if server response with gzipped content, then client will try to unzip them, if not then client will treat as plain data.

So unless we explicitly disable gzip in the client side, this compression will always be on, I'm not sure if it is expected.

This actually can be controlled either in server side or client side, I don't have a strong inclination, just looking for suggestion.